README.md

```diff
@@ -27,14 +31,62 @@ GitHub action to validate CITATION.cff files, and convert to other citation form
          - name: Check out a copy of the repository
            uses: actions/checkout@v2
            
-
          - name: Check whether the citation metadata from CITATION.cff is valid
            uses: citation-file-format/cffconvert-github-action@2.0.0
            with:
              args: "--validate"
            
    ```
 
-1. ``git add``, ``commit`` and ``push`` to GitHub
-1. Check the _Actions_ tab on your repository's page to check the action's output
+### Validating from a subdirectory
+
+   ```yaml
+   name: cffconvert
+   
+   on: push
 
+   jobs:
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+         - name: Validate a CITATION.cff from a subdirectory
+           uses: citation-file-format/cffconvert-github-action@2.0.0
+           with:
+             args: "--infile ./subdirectory/CITATION.cff --validate"
+           
+   ```
+
+### Convert CITATION.cff to Zenodo metadata format
+
+   ```yaml
+   name: cffconvert
+   
+   on: push
+
+   jobs:
+     verify:
+       name: "cffconvert"
+       runs-on: ubuntu-latest
+       steps:
+         - name: Check out a copy of the repository
+           uses: actions/checkout@v2
+           
+         - name: Convert CITATION.cff to Zenodo metadata format
+           id: cffconvert
+           uses: citation-file-format/cffconvert-github-action@2.0.0
+           with:
+             args: "--infile ./CITATION.cff --format zenodo > .zenodo.json"
+            
+        - name: Commit and push Zenodo metadata 
+          run: |
+            git config --global user.name 'cffconvert GitHub Action'
+            git config --global user.email 'cffconvert@users.noreply.github.com'
+            git add .zenodo.json
+            git commit -m "Automated update of Zenodo metadata"
+            git push            
```

Comment:
It this enough? Perhaps it is better to use another action to create a pull request if there is a difference?

Reply:
a good suggestion but this should be added in another PR